README.md

```diff
@@ -7,71 +7,120 @@
+grateful for all of the community help along the way. There's still plenty to do 
+here - and we'd love to have your contributions! Hop on over to the 
+[issues](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues) 
+and have a look around, or open up a 
+[new issue](https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/new).
 
 Features:
 
 * Users can sign in through GitHub Oauth
-* They can create and update teams
-* Teams have members (students, coaches, mentors aka project maintainers), GitHub repositories, a log URL
+* Users can create and update teams (assuming they have the 
+right access rights)
+* Teams have members (students, coaches, mentors aka project maintainers),
+ GitHub repositories, a log URL
 
 * RSS feeds are fetched from all teams' logs regularly and aggregated
-* There is a webhook endpoint for GitHub events that aggregates information about issues, pull requests and such
+* There is a webhook endpoint for GitHub events that aggregates 
+information about issues, pull requests and such
 
-Requirements:
+* Organizers can process new projects for submission, review new teams 
+for selection, keep track of information on conferences for students
+
+Contribution requirements:
 
 * Keep it simple, so Rails Girls students can get involved, too
-* By contributing, you agree to adhere to our [Code of Conduct](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CODE_OF_CONDUCT.md)
+* By contributing, you agree to adhere to our 
+[Code of Conduct](https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/CODE_OF_CONDUCT.md)
 
 ## System Requirements
 
+In order to run the teams application locally so you can contribute, 
+you will need to have the following dependencies already installed. 
+Please see the [wiki](https://github.com/rails-girls-summer-of-code/rgsoc-teams/wiki) 
+for more info on installing these requirements. 
+
 * PostgreSQL 9.3 or newer
 * Ruby 2.4.1
 
-### Setup on Ubuntu
+Instructions are given for Ubuntu or macOS. If you use macOS, we also 
+strongly recommending installing/using [Homebrew](https://brew.sh/) for 
+managing dependencies. 
+
+# Contributing
+
+Thanks so much for your contribution! To get setup to develop the RGSoC 
+teams app, please follow the steps below. Here's some handy links within 
+this section of the README:
+ * [**Quick Start**](#quick-start) - gives a high-level overview 
+of the required steps
+ * [**Detailed Instructions**](#detailed-instructions) - gives 
+this same information, but includes more notes about the individual 
+commands and prompts to expect 
+* [**Optional Configs**](#optional-configs) - gives instructions for
+setting up other tools, such as Google Places API and Mailtrap which 
+may be helpful 
+
+### Quick Start
+0. [Setup System Requirements](#setup-system-requirements)
+1. [Configure Database](#configure-database) - Create a postgres user with superuser rights called `rgsoc` 
```

Comment:
I am _pretty_ sure that Mac users don't need to do this, and it even will be extra complicated if they do. Anyone?